frontend/src/pages/ActivityReport/__tests__/index.js

```diff
@@ -560,134 +560,60 @@ describe('ActivityReport', () => {
         goalsAndObjectives: [],
       });
 
-      fetchMock.get('/api/goals?reportId=1&goalIds=37502', [{
-        endDate: '',
-        status: 'In Progress',
-        value: 37502,
-        label: 'The Grant Recipient will develop a comprehensive plan for staff recruitment, retention and leadership development for all positions',
-        id: 37502,
-        name: 'The Grant Recipient will develop a comprehensive plan for staff recruitment, retention and leadership development for all positions',
-        grant: {
-          programTypes: [],
-          name: 'Barrows Inc - 08bear010431 ',
-          numberWithProgramTypes: '08bear010431 ',
-          recipientInfo: 'Barrows Inc - 08bear010431 - 359',
-          id: 10431,
-          number: '08bear010431',
-          annualFundingMonth: 'November',
-          cdi: false,
-          status: 'Active',
-          grantSpecialistName: 'Marian Daugherty',
-          grantSpecialistEmail: 'Effie.McCullough@gmail.com',
-          programSpecialistName: 'Eddie Denesik DDS',
-          programSpecialistEmail: 'Darryl_Kunde7@yahoo.com',
-          stateCode: 'RI',
-          startDate: '2018-11-01T00:00:00.000Z',
-          endDate: '2023-10-31T00:00:00.000Z',
-          inactivationDate: null,
-          inactivationReason: null,
-          recipientId: 359,
-          oldGrantId: 7764,
-          deleted: false,
-          createdAt: '2021-03-16T01:20:44.754Z',
-          updatedAt: '2022-09-28T15:03:28.432Z',
-          regionId: 1,
-          recipient: {
-            id: 359, uei: 'LS73E9BEHVZ4', name: 'Barrows Inc', recipientType: 'Community Action Agency (CAA)', deleted: false, createdAt: '2021-03-16T01:20:43.530Z', updatedAt: '2022-09-28T15:03:26.284Z',
-          },
-        },
-        objectives: [{
-          id: 95297,
-          label: 'The Grantee Specialists will support the Grant Recipient in reviewing the Planning Alternative Tomorrows with Hope (PATH) 30-Day action items to identify recruitment and retention progress made and celebrate successes.',
-          title: 'The Grantee Specialists will support the Grant Recipient in reviewing the Planning Alternative Tomorrows with Hope (PATH) 30-Day action items to identify recruitment and retention progress made and celebrate successes.',
-          status: 'Not Started',
-          goalId: 37502,
-          resources: [],
-          activityReportObjectives: [],
-          files: [],
+      fetchMock.get('/api/goals?reportId=1&goalTemplateId=24727', [{
+        id: 95297,
```

Comment:
Should this `id` be 24727 to match the query?

Reply:
I don't think so this is returning objectives for a particular report and goal template.